client/js/table-view.js

```diff
@@ -42,6 +44,18 @@ class TableView {
 		this.renderSumRow();
 	}
 
+	addRow(event) {
```

Comment:
Set your text editor to use 2-space tabs instead of 4-space tabs and to “convert indentations to spaces”; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.